intelmqmail/cb.py

```diff
@@ -856,3 +856,7 @@ def mailgen(args, config):
     os.environ['GNUPGHOME'] = config["openpgp"]["gnupg_home"]
 
     mailgen(args, config)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
During refactoring I just kept that possibility to call the script directly.
It is not strictly necessary,
Though it may add to convenience, because you now would need to "pip install"
before getting a callable script and less people would know the python -m call to call it directly.